src/toil/test/cwl/echo_string.cwl

```diff
@@ -0,0 +1,54 @@
+cwlVersion: v1.2
+class: Workflow
+
+inputs:
+  - id: arrayS
+    type: string[]
+    default: ['hello','world']
+
+steps:
+  hello:
+    run:
+      class: CommandLineTool
+      inputs:
+        s: string
+      baseCommand: [echo]
+      arguments: [ $(inputs.s)]
+      outputs:
+        out: stdout
+        err: stderr
+    in:
+      - id: s
+        source: arrayS
+    scatter:
+     - s
+    out:
+      - id: out
+      - id: err
+
+  list:
+    run:
+      class: CommandLineTool
+      inputs:
+       file: File
+      baseCommand: [ ls , -lh]
+      arguments: [ $(inputs.file) ]
+      outputs: []
+    in:
+      - id: file
+        linkMerge: merge_flattened
+        source:
+          - hello/out
+          - hello/err
+    scatter:
+      - file
+    out: []
+
+outputs: []
+
+requirements:
+  - class: ScatterFeatureRequirement
+  - class: MultipleInputFeatureRequirement
+
+$namespaces:
+  cwltool: "http://commonwl.org/cwltool#"
```

Comment:
Does the `test_workflow_echo_string` test (without the associated fix in this PR) still fail if these lines are removed? If so, lets get rid of them. If not, that is very intriguing!

Reply:
Removing only:

```
$namespaces:
  cwltool: "http://commonwl.org/cwltool#"
```

This CWL workflow will still fail with the same predecessor error (without the fix).  I've removed it for the test.

Reply:
I'm going to do a little more work investigating this.

Reply:
Okay, then I think it is fine to remove it.